packages/cardano-services/src/StakePoolSearch/DbSyncStakePoolSearchProvider/queries.ts

```diff
@@ -576,7 +576,9 @@ SELECT
   pu.vrf_key_hash,
   metadata.url AS metadata_url, 
   metadata.hash AS metadata_hash,
-  pod.json AS offline_data
+  pod.json AS offline_data,
+  pod.json -> 'name' AS name,
+  ((pu.fixed_cost * pu.margin) + pu.fixed_cost) as total_cost
```

Comment:
Not sure if that calculation is correct, refer to https://input-output.atlassian.net/browse/ADP-1755 latest comment